### PR TITLE
New File Added: Assi5,assi6,assi9

### PR DESCRIPTION
**Pull request by vinod2406**

### CODE REVIEW

File: vinod/javascript/ASSIN5/ass5js1.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                    <td><input type="text" name="a" id="first" placeholder="Enter a number" /></td>
+                </tr>
+                <tr>
+                    <td><button onclick="middle ()">Submit</button></td>
+                </tr>
+            </table>
+            <div id="num"></div>
+        </div>
+    </body>
+
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN5/ass5js1.html]: Remove extra line.

File: vinod/javascript/ASSIN5/ass5.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN5/ass5.html]: This style should be present within head. Make this changes everywhere.

File: vinod/javascript/ASSIN5/ass5js1.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            first = parseInt(n / 100);
+            sum = last + first;
+            if (sum == middle) {
+                document.getElementById("num").innerHTML = "the sum is matching";
+            } else {
+                document.getElementById("num").innerHTML = "the sum is not matchinng";
+            }
+        }
+    </script>
+</html>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN5/ass5js1.html:48]: Closing body tag is always close before the closing HTML tag.

File: vinod/javascript/ASSIN5/ass5js3.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN5/ass5js3.html]: This style should be present within head

File: vinod/javascript/ASSIN5/ass5js4.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN5/ass5js4.html:16]: This style should be present within head

File: vinod/javascript/ASSIN5/ass5js4.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                </tr>
+                <tr>
+                    <td><button onclick="sum_product6 ()">Submit</button></td>
+                </tr>
+            </table>
+            <div id="num"></div>
+            <div id="num1"></div>
+        </div>
+    </body>
+
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN5/ass5js4.html]: remove extra line.

File: vinod/javascript/ASSIN6/ass1q1.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                width: 433px;
+                height: 575px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: -21px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN6/ass1q1.html]: This style should be present within head

File: vinod/javascript/ASSIN6/ass1q1.html [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+        <div class="as">
+                       <h1> name number list:</h1>
+                        First Name<input type="text" id="inp1" class="form-control" /> Second Name<input type="text" id="inp2" class="form-control" /> Third
+                        Name<input type="text" id="inp3" class="form-control" /> Fourth Name<input type="text" id="inp4" class="form-control" /> Fifth
+                        Name<input type="text" id="inp5" class="form-control" />
+                        <button onclick="fun()" class=>Click Me</button>
+                        <p name="" id="show" ></p>
+        </div>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN6/ass1q1.html:44]: give indentation properly.

File: vinod/javascript/ASSIN6/ass6.q2.html [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN6/ass6.q2.html]: This style should be present within head

File: vinod/javascript/ASSIN6/ass6.q4.html [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
+        <div class="as">
+            <h1>Sum of all factorial !!</h1>
+
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN6/ass6.q4.html]: remove extra line.

File: vinod/javascript/ASSIN6/ass6.q4.html [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN6/ass6.q4.html]: This style should be present within head.

File: vinod/javascript/ASSIN6/ass6jsq3.html [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN6/ass6jsq3.html:20]: This style should be present within head.

File: vinod/javascript/ASSIN6/ass6jsq5.html [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN6/ass6jsq5.html:20]: This style should be present within head.

File: vinod/javascript/ASSIN9/ass9q3.html [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+                width: 568px;
+                height: 610px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN9/ass9q3.html]: This style should be present within head.

File: vinod/javascript/ASSIN9/ass9q5.html [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+                width: 340px;
+                height: 415px;
+                margin: 51px auto 0px auto;
+                border-style: solid;
+                top: 22px;
+                left: 399px;
+                margin-right: 10px;
+                background-color: darkturquoise;
+            }
+        </style>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN9/ass9q5.html:20]: This style should be present within head.

File: vinod/javascript/ASSIN9/ass9q6.html [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+                    <div>
+                        <p>
+                            Total word &nbsp; = &nbsp;<span id="num1"></span> <br /><br />
+                            Whitespaces &nbsp; = &nbsp;<span id="num2"></span>
+                        </p>
+                    </div>
+                    <div>
+                        <button class="btn1" onclick="centence()">Check</button>
+                    </div>
+        </div>
```
Comment (Pranjali1999) [inline vinod/javascript/ASSIN9/ass9q6.html]: give indentation properly.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

**Review by Pranjali1999 — APPROVED** [review]

File: vinod/javascript/ASSIN6/ass1q1.html [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+    <body>
+        <div class="as">
+                       <h1> name number list:</h1>
+                        First Name<input type="text" id="inp1" class="form-control" /> Second Name<input type="text" id="inp2" class="form-control" /> Third
+                        Name<input type="text" id="inp3" class="form-control" /> Fourth Name<input type="text" id="inp4" class="form-control" /> Fifth
+                        Name<input type="text" id="inp5" class="form-control" />
+                        <button onclick="fun()" class=>Click Me</button>
+                        <p name="" id="show" ></p>
+        </div>
+    </body>
```
Comment (Pawan1005) [inline vinod/javascript/ASSIN6/ass1q1.html:45]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**